ci/gpu/build.sh

```diff
@@ -61,7 +61,9 @@ conda install -c conda-forge -c rapidsai -c rapidsai-nightly -c nvidia \
       "ucx-py=${MINOR_VERSION}" \
       "statsmodels" \
       "xgboost====1.0.2dev.rapidsai0.13" \
-      "lightgbm"
+      "lightgbm" \
+      "clang=8" \
+      "clang-tools=8"
```

Comment:
With the changes in rapidsai/gpuci-build-environment#68 and rapidsai/integration#26 these won't be necessary

Reply:
Thanks. Makes sense. Fixed this one.